core/field_textinput.js

```diff
@@ -106,6 +107,20 @@ Blockly.FieldTextInput.prototype.CURSOR = 'text';
  */
 Blockly.FieldTextInput.prototype.spellcheck_ = true;
 
+/**
+ * Configure the field based on the given map of options.
+ * @param {Object} opt_config A map of options to configure the field based on.
+ * @private
+ */
+Blockly.FieldTextInput.prototype.configure_ = function(opt_config) {
+  if (!opt_config) {
+    return;
+  }
+  if (typeof opt_config['spellcheck'] == 'boolean') {
+    this.setSpellcheck(opt_config['spellcheck']);
```

Comment:
You're calling setSpellcheck here, but this in inconsistent with what is being done elsewhere (eg: [checkbox field](https://github.com/google/blockly/blob/develop/core/field_checkbox.js#L142)). Why is that?

Reply:
No reason, just a derp on my part. Thanks for the catch!